webserver/views/api/v1/datasets.py

```diff
@@ -217,7 +217,18 @@ def add_recordings(dataset_id):
 
     :resheader Content-Type: *application/json*
     """
-    raise NotImplementedError
+    class_dict = request.get_json()
+    if not class_dict:
+        raise api_exceptions.APIBadRequest("Data must be submitted in JSON format.")
+    try:
+        db.dataset.add_recordings(class_dict, dataset_id)
```

Comment:
This doesn't check who the owner of the dataset is, which means that anyone can modify someone else's dataset. You should use the `get_check_dataset` method to perform this check.